bonxai_core/src/vector_output_stream.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Contributors to the Bonxai Project
+ * Copyright Contributors to the OpenVDB Project
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <ostream>
+#include <streambuf>
+#include <vector>
+
+namespace Bonxai {
+class VectorStreamBuffer final : public std::streambuf {
```

Comment:
add documentation

Reply:
Can do but this is an implementation detail, not sure if it needs _much_ documentation.

Reply:
Fixed